code/src/App.js

```diff
@@ -3,14 +3,17 @@ import React, { useState, useEffect } from 'react';
 import ThoughtList from 'components/ThoughtList';
 import ThoughtForm from 'components/ThoughtForm';
 
+// const API_URL = 'https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/'
+const API_URL = 'https://project-happy-thoughts-api-ircjrh2jfq-lz.a.run.app/'
+
```

Comment:
Very nice to use the const API_URL for the link, to eliminate problems with the link.
And to include / in the API to only have the use the endpoint.
I guess the only change you made in frontend is to replace the link.